### ISSUE TITLE
Support SDK-based projects

### ISSUE THREAD

**Issue by 3F** [3F/MvsSln#2]
The bug was described here: https://github.com/3F/DllExport/issues/56

For example, the following definition below should cause a problem as for #1:

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp2.0</TargetFramework>
  </PropertyGroup>
</Project>
```
```
The default XML namespace of the project must be the MSBuild XML namespace or no namespace.
```

**Comment by 3F** [comment]
Same here, https://github.com/3F/MvsSln/issues/5#issuecomment-410464661 solves problem. But this only for .netfx 4.6

**Comment by SH2015** [comment]
Any work around about this ?  I am getting error with SDK style .netframework  project

**Comment by 3F** [comment]
Yes, https://github.com/3F/MvsSln/issues/23#issuecomment-639147675

Unfortunately I can't personally continue yet due to https://twitter.com/GitHub3F/status/1302228682870841345

For 2.x please follow the information here https://github.com/3F/MvsSln/wiki/Advanced-Features#how-to-use-modern-versions-of-msbuild-for-netfx

Thanks for using! Follow the news.

**Comment by SH2015** [comment]
@3F  thanks and hope to get better
